history-service/server.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const { Pool } = require('pg');
+const cors = require('cors');
+
+const service_port = 5433;
+const sql_port = 5432;
+
+const app = express();
+const pool = new Pool({
```

Comment:
Just curious, did you consider using Sequelize to help with defining the model?

Reply:
Would also be much easier to use Sequelize instead of writing raw SQL queries

Reply:
I feel Sequelize is hard to test and work with from external SQL controller